src/postprocessing/idlparsed.js

```diff
@@ -8,17 +8,170 @@
 const webidlParser = require('../cli/parse-webidl');
 
 module.exports = {
-  dependsOn: ['idl'],
+  dependsOn: ['dfns', 'idl'],
   input: 'spec',
   property: 'idlparsed',
 
   run: async function(spec, options) {
+    function getHref(idl, member) {
+      let dfnType;
+      let dfnFor;
+      let dfnOverload = 0;
+      let dfnName;
+      if (member) {
+        if (['iterable', 'maplike', 'setlike'].includes(member.type) ||
+            ['getter', 'setter', 'stringifier', 'deleter'].includes(member.special)) {
+          // No dfns of these types in any spec as of Feb 2024, or at least no
+          // no dfns that we can easily map to (for example, the HTML spec
+          // tends to use generic "dfn" for these).
+          return null;
+        }
+        if (member.type === 'operation') {
+          dfnType = 'method';
+          dfnOverload = idl.members
```

Comment:
I think this should distinguish static operations from non-static ones (since they create different overload sets)